packages/playwright/src/reporter.ts

```diff
@@ -108,6 +108,7 @@ class ReplayPlaywrightReporter implements Reporter {
   > = {};
 
   constructor() {
+    addReplayFixture();
```

Comment:
this seems to work when called from here but the question is - was there a specific reason why it wasn't called from here?

Reply:
Cool, it's good to at least have some confirmation that there is no obvious blocker to move it here 👍